labeled_data/companies/alibaba/index.yml

```diff
@@ -50,5 +50,4 @@ data:
     - 102641885 # org:alibaba-open-source-website
     - 101698007 # org:koordinator-sh
     - 100843567 # org:opensergo
-    - 90233428 # org:opensumi
```

Comment:
Thanks for the data update, are you sure to remove opensumi from alibaba? We can add opensumi to both alibaba and antgroup like seata.

Reply:
![CleanShot 2022-09-29 at 23 10 25@2x](https://user-images.githubusercontent.com/13938334/193069664-a7d200fc-b61a-44d8-9cb8-5429cf114f4f.png)

Maybe it's my fault, so I need add opensumi to the label field?

Reply:
Yes, you should add `opensumi` to `alibaba/index.yml`'s label field like other projects.